make_var_mx6ul_dart_debian.sh

```diff
@@ -451,7 +451,7 @@ EOF
 
 	pr_info "rootfs: install selected debian packages (third-stage)"
 	chmod +x third-stage
-	LANG=C chroot ${ROOTFS_BASE} /third-stage
+	LANG=C chroot ${ROOTFS_BASE}/third-stage
```

Comment:
This will not run third_stage, but instead, it will try to open /bin/sh using ${ROOTFS_BASE}/third-stage as root filesystem

Reply:
I got a bunch of errors when the script got to third stage about it being unable to find `apt-get` so I launched the `third-stage` script separately and it worked fine. Then went fixed this path in the script above and ran it without any problems... but I see what you're saying, I might be wrong with that.

Reply:
👍 I'll just wait for one of the maintainer's feedback

Reply:
@stdcerr we tested your suggestion, but it doesn't produce any of the error mentioned above. Could you please share your complete build details and build machine details? It seems like you may be using a different environment. 
We do all tests on the builds and verify the build before publishing the release/changes. 
You can always log your question/release on our customer portal support. 
Variscite customer portal is our official place for technical questions and you will get faster answers there.
https://varisciteportal.axosoft.com/

Reply:
@stdcerr 
I am unable to re-produce the issue. 
It seems your build environment could be the factor. 
I am closing this issue for now. If you have further questions, please create a ticket at https://varisciteportal.axosoft.com/